src/components/UI/Display/Apps/AppList/AppsListPage.tsx

```diff
@@ -11,7 +11,9 @@ import Toolbar from './Toolbar';
 
 const LOADING_COMPONENTS = new Array(9).fill(0);
 
-const Wrapper = styled.div``;
+const Wrapper = styled.div`
+  margin-top: -10px;
```

Comment:
Is it possible to not have negative margin here? As I understand it is here to compensate `<AppInstallationSelectedCluster/>` component's `margin-top`, right? But maybe we can just remove `top` property [here](https://github.com/giantswarm/happa/pull/3338/files#diff-4b91d69bf401498c7d748bed07fabed1e6dbc800b819e880d255bd65811aa580R314)?

Reply:
I used it to try and even out the vertical margins when there's no selected cluster banner - this is how it would look without the negative margin:
<img width="500" alt="Screen Shot 2022-04-13 at 15 43 39" src="https://user-images.githubusercontent.com/62935115/163194413-a4c6fa90-e9e8-4f76-b13d-2c9c43efcb03.png">
The position of `<Wrapper>` is influenced by the padding on the `<main>` element (120px).

Another approach I was thinking of was to increase the bottom margin of the text instead (to 30px), but only when there's no selected cluster banner:
<img width="500" alt="Screen Shot 2022-04-13 at 15 55 35" src="https://user-images.githubusercontent.com/62935115/163197287-0775f534-9d58-4879-913b-2298f6a3574b.png">
Removing the `margin-top` here:
<img width="500" alt="Screen Shot 2022-04-13 at 15 57 41" src="https://user-images.githubusercontent.com/62935115/163197291-6ce262ea-261f-47c2-a3a8-82a7b750d747.png">

What do you think?